compiler/rustc_error_messages/locales/en-US/monomorphize.ftl

```diff
@@ -0,0 +1,19 @@
+monomorphize_recursion_limit =
+    "reached the recursion limit while instantiating `{$shrunk}`
```

Comment:
```suggestion
    reached the recursion limit while instantiating `{$shrunk}`
```

Reply:
Ha! I fixed this and forgot to push it up.